docs/en/api/query-protocol.md

```diff
@@ -189,10 +189,12 @@ extend type Query {
 Event query fetches the event list based on given sources and time range conditions.
 
 ### Profiling
-SkyWalking offers two types of [profiling](../concepts-and-designs/profiling.md), in-process and out-process, allowing users to create tasks and check their execution status.
```

Comment:
We still only have two kinds of profiling. Could you put async profiler in the current catalog? Is it running in-process or out-process?